### ISSUE TITLE
Incorrect mask for Bangladesh

### ISSUE THREAD

**Issue by alexandrchebotar** [ChristoPy/countries-phone-masks#1]
```
{
    "name": "Bangladesh",
    "code": "+880",
    "iso": "BD",
    "flag": "https://www.countryflags.io/BD/flat/24.png",
    "mask": "##-###-###"
  },
```
There should be 10 digits. https://en.wikipedia.org/wiki/Telephone_numbers_in_Bangladesh

**Comment by ChristoPy** [comment]
Hi, sorry for late response!

Thanks for the contribution, I'll release a for for it.

**Comment by ChristoPy** [comment]
Fixed on #2 ,

Feel free to make more contributions!

Thanks again.
